spyder/utils/qthelpers.py

```diff
@@ -499,6 +499,10 @@ def show_std_icons():
     dialog.show()
     sys.exit(app.exec_())
 
+    
+def menu_separator():

```

Comment:
Let's make this one a constant instead of a function to use it more easily, like this

menu_separator = None